fishtest/fishtest/rundb.py

```diff
@@ -194,7 +194,6 @@ def get_results(self, run):
 
     run['results_stale'] = False
     run['results'] = results
-    self.runs.save(run)
```

Comment:
This is not related to update_task. 

It stores the result in the database, I think we should keep this line.

Reply:
Hmmm, it's called at line 307 `sprt_stats = stat_util.SPRT(self.get_results(run)` so I can see why I wanted to remove the save, but it's also called a lot from views.py so you're right, the current change is no good.

I could add a boolean 
```
def get_results(self, run, save_run = True):
...
  if save_run:
    self.runs.save(run)
```
 and modify the call in update_task() to pass `False` in. That way we remove the save from update_task() without affecting the other calls?

Reply:
Sounds good